histogram.py

```diff
@@ -145,18 +146,26 @@ def render_message(table, message):
 def render(table, params):
     column = params["column"]
     if not column:
-        return render_message(table, "Please choose a number column")
+        return render_message(
+            table, trans("errors.noColumnSelected", "Please choose a number column")
+        )
 
     raw_series = table[column]
     if not is_numeric_dtype(raw_series):
-        return render_message(table, "Please choose a number column")
+        return render_message(
```

Comment:
This `errors.columnType` is cruft. I've updated `histogram.json` to disallow non-numeric data; now this condition is deleted on master.